static/src/stylesheets/module/content/_article-minute.scss

```diff
@@ -571,8 +571,12 @@ $minute-layout-margin-nudge: gs-span(1);
                     width: 100%;
                 }
 
-                .gu-image {
+                .element--inline__outer {
```

Comment:
This is the (simplified) markup: 

```
<figure class="element element--inline">
  <div class="element--inline__outer"></div>
</figure>
```

`outer` depends on `element--inline` and shouldn't be used anywhere else. Since it's a child element rather than a modifier, shouldn't it be separated by underscores?